exonum/src/helpers/mod.rs

```diff
@@ -60,3 +60,42 @@ pub trait ValidateInput: Sized {
 pub fn byzantine_quorum(total: usize) -> usize {
     total * 2 / 3 + 1
 }
+
+/// Module for serializing `Option<Hash>` to Protobuf.
```

Comment:
Crap, the option should most probably be `ignore` instead of `no_run` (`no_run` compiles the code snippet, which requires the Protobuf source structure to exist).

Reply:
No problem, fixed.